docs-source/content/userguide/managing-domains/model-in-image/usage.md

```diff
@@ -0,0 +1,173 @@
+| -------------------------                    |  ------------------ |
+| `domainHomeSourceType`                       |  Required. Set to `FromModel`. |
+| `domainHome`                                 |  Must reference an empty or non-existent directory within your image. Do not include the mount path of any persistent volume. Note that Model in Image recreates the domain home for a WebLogic pod every time the pod restarts.|
+| `configuration.model.configMap`             | Optional. Set if you have stored additional models in a config map as per [Optional WDT model config map](#3-optional-wdt-model-config-map). |
+| `configuration.secrets`                      | Optional. Set this array if your image or config map models contain macros that reference custom Kubernetes secrets. For example, if your macros depend on secrets `my-secret` and `my-other-secret`, then set to `[my-secret, my-other-secret]`.|
+| `configuration.model.RuntimeEncryptionSecret`| Required. All Model in Image domains must specify a runtime encryption secret. See [Required runtime encryption secret](#4-required-runtime-encryption-secret). |
+| `configuration.model.domainType`             | Set the type of domain. Valid values are `WLS`, `JRF`, and `RestrictedJRF` where `WLS` is the default. See [WDT Domain Types](https://github.com/oracle/weblogic-deploy-tooling/blob/master/site/type_def.md).|
+
+Notes:
+
+ - There are additional attributes that are common to all domain home source types, such as the `image` field. See the Domain Resource [schema](https://github.com/oracle/weblogic-kubernetes-operator/blob/master/docs/domains/Domain.md) and [documentation]({{< relref "/userguide/managing-domains/domain-resource.md" >}}) for a full list of domain resource fields.
+
+ - There are also additional fields that are specific to JRF domain types. For more information, see [Prerequisites for JRF domain types](#7-prerequisites-for-jrf-domain-types).
+
+ - Sample domain resource: For an example of a fully specified sample domain resource, see the the operator source's `kubernetes/samples/scripts/create-weblogic-domain/model-in-image/k8s-domain.yaml.template` file for the [Model in Image]({{< relref "/samples/simple/domains/model-in-image/_index.md" >}}) sample. Note that the `@@` entries in this template are not processed by the operator; they need to replaced with actual values before the resource can be applied.
+
+#### 7. Prerequisites for JRF domain types
+
+{{% notice info %}} This section applies only for a `JRF` domain type. Skip it if your domain type is `WLS` or `RestrictedJRF`.
+{{% /notice %}}
+
+A JRF domain requires an infrastructure database called an RCU database, initializing this database, and configuring your domain to access this database. All of these steps must occur before you first deploy your domain. When you first deploy your domain, the introspector job will initialize it's RCU schema tables in the database - a process that can take several minutes.
```

Comment:
This is not entirely correct.  The operator keeps a copy of the wallet file in the introspector configmap.  The wallet file is only required if you want to share the database with other domains or if the domain is deleted and recreated (essentially sharing with another domain).  The reason for this is - we split into primordial domain and then update with the rest of the config.  The primordial jrf domain has all the wallet and all the default JRF apps/resources info. In order to re-create another domain to share the rcu database,  the wallet+password must be provided.  We only re-create the primordial domain if there is a security info change (e.g. wls credentials and security roles) and for the same domain if the walletFile is not provided, it will reuse the one in the introspect configmap.   

 However, for safety and best practice, user should save , backup and use the walletFile in the domain resource yaml - such as the case when the domain is deleted by accident.

Reply:
It looks right to me - let's discuss tomorrow.  Once a domain is deployed, the db is initialized.   We don't want to give customer a false sense of security that since their wallet happens to be in a config map then 'it's safe'.  It's not safe.  If _anything_ goes wrong - power outage, crash, etc, etc, then the config map is lost and the wallet is lost.    As for your 'However,' the documentation after the above covers the wallet recovery use case.

Reply:
Oh. You were actually talking about the following paragraph.  I've updated the wording slightly so that simply makes sure one knows that one must *always* save the wallet after a first starting a domain, without explaining that the wallet stays around for a while in an introspect map for re-use if you happen to restart the domain without losing the introspector config map in between.   Please take a look and comment.  Or resolve if yo uthink the change is good.

Reply:
@jshum2479 Please comment, or resolve if you think this is good now.

Reply:
Looks good